compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/PlistSanityCheck.uikit.kt

```diff
@@ -37,7 +37,21 @@ internal object PlistSanityCheck {
                     .objectForInfoDictionaryKey("CADisableMinimumFrameDurationOnPhone") as? NSNumber
 
                 if (entry?.boolValue != true) {
-                    println("WARNING: `Info.plist` doesn't have a valid `CADisableMinimumFrameDurationOnPhone` entry. Framerate will be restricted to 60hz on iPhones. To support high frequency rendering on iPhones, add `<key>CADisableMinimumFrameDurationOnPhone</key><true/>` entry to `Info.plist`.")
+                    if (enforceStrictPlistSanityCheck) {
+                        val message = """
+                            Error: `Info.plist` doesn't have a valid `CADisableMinimumFrameDurationOnPhone` entry. 
+                            This will result in an inadequate performance on devices with high refresh rate. 
+                            Add `<key>CADisableMinimumFrameDurationOnPhone</key><true/>` entry to `Info.plist` to fix this error. 
+                            If you don't have a separate plist file, add the entry to the target from within Xcode: Project -> Targets -> Info -> Custom iOS Target Properties.                            
+                            Or set `ComposeUIViewControllerConfiguration.enforceStrictPlistSanityCheck` to false, if it's intended.                            
+                        """.trimIndent()
+                        println(message)
+                        terminateWithUnhandledException(
+                            IllegalStateException(message)
+                        )
```

Comment:
```suggestion
                        error(message)
```

`terminateWithUnhandledException` is native only and `ExperimentalNativeApi`. `error` on the other hand is common and established way to tell user that is something wrong.

Unless I don't know something about `terminateWithUnhandledException`, let's use `error`